pandas/util/testing.py

```diff
@@ -1306,33 +1306,36 @@ def assert_frame_equal(left, right, check_dtype=True,
                        check_categorical=True,
                        check_like=False,
                        obj='DataFrame'):
-    """Check that left and right DataFrame are equal.
+    """
+    Check that left and right DataFrame are equal.
 
     Parameters
     ----------
     left : DataFrame
+        First frame to compare.
     right : DataFrame
+        Second frame to compare.
     check_dtype : bool, default True
         Whether to check the DataFrame dtype is identical.
-    check_index_type : bool / string {'equiv'}, default False
+    check_index_type : bool / string {'equiv'}, default 'equiv'
```

Comment:
I think what we've been using in these cases is `{'equiv'} or bool`

Reply:
IMO that is more confusing. e.g. it might imply I pass a set. I'm not sure it's worth a special case for when there is only one possible string value.

Reply:
what is your suggestion, just `'equiv' or bool, default 'equiv'` or something else? Besides being more consistent for the user, using the curly brackets in all cases would simplify parsing the types and adding validation and extracting stats. But if you are strongly in favor of not using them, I'm happy to merge this with it now, and see later on what's best when we implement that validation.

Reply:
I think as it is above is good. It's maybe a little verbose but is very clear.

```check_index_type : bool / string {'equiv'}, default 'equiv'```

Happy to revisit if a standard emerges.

Reply:
You have the standard here: https://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html#parameter-types

If you can use the first format I suggested, that would be great.

Reply:
Thanks. I have used your suggestion.